src/api/exceptions.cc

```diff
@@ -41,8 +56,7 @@ Local<Value> ErrnoException(Isolate* isolate,
 
   Local<String> path_string;
   if (path != nullptr) {
-    // FIXME(bnoordhuis) It's questionable to interpret the file path as UTF-8.
-    path_string = String::NewFromUtf8(isolate, path).ToLocalChecked();
+    path_string = StringFromPath(isolate, path);
```

Comment:
This FIXME should not be removed, isn't it?

Reply:
I think this dates waaay back and it's possible that some other path is passing a wide string from Windows to this that's not originated from JS but from some Windows API. So this should probably still be left as-is.